test/e2e/oap/Dockerfile.oap

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk:8-jdk
+
+ARG COMMIT_HASH=408bbf86e6bc75f6a95f11c346daecd570f6b45e
+
+WORKDIR /sw_code
+
+RUN apt-get update && apt-get install -y make git netcat
+RUN git clone https://github.com/apache/skywalking $(pwd) && git reset --hard ${COMMIT_HASH}
+RUN export SKIP_TEST=true && make init && make build.all
+
+RUN set -ex; \
+    cd dist; \
+    tar -xzf "apache-skywalking-apm-bin.tar.gz"; \
+    rm -rf "apache-skywalking-apm-bin.tar.gz"; \
+    rm -rf "apache-skywalking-apm-bin/bin"; rm -rf "apache-skywalking-apm-bin/webapp"; rm -rf "apache-skywalking-apm-bin/agent"; \
+    rm -rf "apache-skywalking-apm-bin/config/log4j2.xml"; \
+    mkdir "apache-skywalking-apm-bin/bin"; \
+    mv "apache-skywalking-apm-bin" /skywalking;
+
+WORKDIR /skywalking
+
+COPY log4j2.xml config/
+COPY docker-entrypoint.sh .
```

Comment:
For `log4j2.xml` we might need to customized it in test case, but for `docker-entrypoint.sh`, I think we can simply reuse the one cloned in the previous steps, line 24.

Reply:
Sure, for `log4j2.xml` maybe I just need to delete some `Logger` declare? for the `docker-entrypoint`, I will just copy from the main repo.

Reply:
> Sure, for `log4j2.xml` maybe I just need to delete some `Logger` declare?

I mean this file can be replicated here, with or without modifications (as per your need).

> for the `docker-entrypoint`, I will just copy from the main repo.

Yes.